cmd/minishift/cmd/stop.go

```diff
@@ -65,6 +66,6 @@ func runStop(cmd *cobra.Command, args []string) {
 }
 
 func init() {
-	stopCmd.Flags().BoolVar(&skipUnRegistration, "skip-unregistration", false, "Skip the virtual machine unregistration.")
+	stopCmd.Flags().BoolVar(&cmdUtil.SkipUnRegistration, "skip-unregistration", false, "Skip the virtual machine unregistration.")
```

Comment:
👎 this should be in a registration package, such as `cmdRegistration`, not `cmdUtil`.